runtime/src/lib.rs

```diff
@@ -351,7 +351,7 @@ impl pallet_gear::Config for Runtime {
     type OutgoingLimit = ConstU32<1024>;
     type DebugInfo = DebugInfo;
     type CodeStorage = GearProgram;
-    type MailboxThreshold = ConstU64<0>;
+    type MailboxThreshold = ConstU64<3000>;
```

Comment:
How did you choose this value?

Reply:
Random for now.
This needs to be discussed/benchmarked somehow before release